File: WEB-API/WEB-API/Models/Predmet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WEB_API.Models
{
    public class Predmet
    {
        public virtual int IdPredmet { get; set; }
        public virtual string Naziv { get; set; }
        public virtual Smer Smer { get; set; }
        public virtual GodinaStudija Godina { get; set; }

        
    }
}

File: AB-ArhivaBlanketa/app/src/main/java/com/ab/ssd/ab_arhivablanketa/Klase/Predmet.java
package com.ab.ssd.ab_arhivablanketa.Klase;


import java.io.Serializable;

public class Predmet implements Serializable {
    public int ID ;
    public String naziv ;
    public Smer smer ;
    public GodinaStudija godinaStudija;

    public Predmet(){}
    public Predmet(int i, String n, Smer s, GodinaStudija g)
    {
        this.ID=i;
        this.naziv=n;
        this.smer=s;
        this.godinaStudija=g;
    }

    public String toString()
    {
        return naziv;
    }
}
